docs/qasm.rst

```diff
@@ -121,3 +121,26 @@ standard gates and other included file is sufficient. However, note that a
 Braket.
 
 .. autofunction:: mpqp.qasm.qasm_to_braket.qasm3_to_braket_Program
+
+From OpenQASM to Cirq
+---------------------
+
+The Cirq library allows the user to instantiate a Cirq ``Circuit`` from an
+OpenQASM 2.0 code. This feature enables users to seamlessly transition 
+their quantum circuits from OpenQASM to Cirq, leveraging Cirq's capabilities
+and features.
+
+the Cirq parser lacks native support for certain OpenQASM 2.0 operations such as
+``cu1``, ``crz``, ``cu3``, ``reset``, ``u0``, ``p``, ``cp``, ``u``, ``rzz``, 
+``rxx`` and custom ``gate``. 
+To address this limitation, we've temporarily incorporated these gates directly 
+into the parser and for the custom gate we replace directly by native gate in 
```

Comment:
The cirq parser or our parser ?
If it is the cirq parser, how can we profit from those changes ?